tests/bluetooth/tester/overlay-bt_ll_sw_split.conf

```
@@ -0,0 +1,75 @@
+# This file is designed to be used with the BT Tester with the Zephyr Bluetooth LE Controller using
+# BT_LL_SW_SPLIT.
+# It is designed to be board-independent (assuming that it fits in memory).
+
+CONFIG_LTO=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+
+# Host and Controller common dependencies
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_PER_ADV_SYNC_TRANSFER_SENDER=y
+CONFIG_BT_PER_ADV_SYNC_TRANSFER_RECEIVER=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_ISO_BROADCASTER=y
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_CENTRAL=y
+CONFIG_BT_ISO_PERIPHERAL=y
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG=y
+CONFIG_MBEDTLS_AES_ROM_TABLES=y
+
+# Generic Controller configurations
+CONFIG_BT_LL_SW_SPLIT=y
+
+# BAP may require up to 2 channels of 155 octets per frame => 310 SDU size required
+CONFIG_BT_ISO_TX_MTU=310
+CONFIG_BT_ISO_RX_MTU=310
+CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=310
+
+# In theory, CONFIG_BT_ISO_TX_BUF_COUNT=1, should be sufficient but this count
+# is used in the context of IPC which falls into a "Newton's Cradle" effect
+# where probably (CONFIG_BT_CTLR_ISO_TX_BUFFERS - CONFIG_BT_ISO_TX_BUF_COUNT)
+# buffers get throttled. Hence, always have the value equal or greater.
+CONFIG_BT_ISO_TX_BUF_COUNT=8
+
+# CSIP tests requires up to 3 devices
+CONFIG_BT_MAX_CONN=3
+CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=3
+
+# BAP/BSRC/SCC/BV-38-C requires 2 BIGs
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+CONFIG_BT_ISO_MAX_BIG=2
+
+# Multiple BAP tests requires 2 ISO channels for both unicast and broadcast
+CONFIG_BT_ISO_MAX_CHAN=4
+CONFIG_BT_CTLR_ISOAL_SOURCES=4
+CONFIG_BT_CTLR_ISOAL_SINKS=4
+CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_MAX=2
+CONFIG_BT_CTLR_CONN_ISO_STREAMS_PER_GROUP=2
+
+# Support both
+CONFIG_BT_CTLR_ADV_ISO_SEQUENTIAL=y
+CONFIG_BT_CTLR_ADV_ISO_INTERLEAVED=y
+
+# Set TX buffers so that we have at least 3 buffers per channel
+# in the Zephyr Bluetooth Controller implementation each SDU is buffered for up to 2 ISO interval:
+# - One ISO interval until transmission on-air,
+# - One ISO interval thereafter to generate Number of Completed packets,
+# - One additional Tx buffer for upper layer to enqueue new SDU.
+CONFIG_BT_CTLR_ISO_TX_BUFFERS=12
```

```suggestion
CONFIG_BT_CTLR_ISO_TX_BUFFERS=6
```

This does inherit from `CONFIG_BT_ISO_TX_BUF_COUNT`. You may decide to comment out this assignment.

---

I've removed `BT_ISO_TX_BUF_COUNT` and only assign `CONFIG_BT_CTLR_ISO_TX_BUFFERS`. 

The reason for this is that `BT_ISO_TX_BUF_COUNT` is  only used in the host, and the number of buffers the host uses is different. Since this file is also planned to be used for the nrf52bsim build, we don't want/need to set the host buffer count here as well.

---

Wait... `CONFIG_BT_ISO_TX_BUF_COUNT` has to match on both Host and hci_ipc side too, otherwise there will be SDUs being dropped in IPC (unless that has been fixed).